email/1.3/modules/ROOT/pages/email-list.adoc

```diff
@@ -1,203 +1,209 @@
 = List Emails with Email Connector Examples - Mule 4
 :page-aliases: connectors::email/email-list.adoc
 
-Email Connector can list emails from any part of the flow. This capability is a change from the Mule 3 transport, which could retrieve emails only as a result of inbound endpoint polling.
+Anypoint for Email (Email Connector) provides the *List - IMAP* and *List - POP3* operations to list emails from any part of the flow. The following examples illustrate how to configure these operations for the connector.
+
+* <<send-attachments,List Emails from an IMAPS Server>> +
+Configure the *List - IMAP* operation to list all the emails in the configured IMAP mailbox folder from an IMAPS server with TLS connection. Additionally, set a *For Each* scope and *Set Variable* components to iterate over the emails and save the email body and attachment content in variables.
```

Comment:
The end of the sentence is not clear to me. Is it the IMAPs server that has a TLS connection? A TLS connection to what?

components > component